genesis/process/shardGenesisBlockCreator.go

```diff
@@ -40,6 +40,22 @@ type deployedScMetrics struct {
 	numOtherTypes int
 }
 
+func createGenesisConfig() config.GeneralSettingsConfig {
+	return config.GeneralSettingsConfig{
+		BuiltInFunctionsEnableEpoch:            0,
+		SCDeployEnableEpoch:                    unreachableEpoch,
+		RelayedTransactionsEnableEpoch:         0,
+		PenalizedTooMuchGasEnableEpoch:         0,
+		AheadOfTimeGasUsageEnableEpoch:         unreachableEpoch,
```

Comment:
we will have to be carefull at hardfork :)

Reply:
I did some digging in the code and found out that, in case of hardfork event, the hardfork processors will use the provided configuration (from the files), not this mocked config. So, I think we are safe to have this as it is.